### PR TITLE
AB#8535 Tabs header

### PR DESCRIPTION
**Pull request by sc-nelikostadinova**

### CODE REVIEW

File: play-media-native/hooks/useFacets/useFacets.ts [hunk 1]
```diff
@@ -32,3 +33,27 @@ export const useFacets = ({
...
+export const useSearchFacets = ({
+  initialItems,
+  facets,
+  query,
+}: {
+  initialItems: Record<string, any>[];
+  facets: IIndexable;
+  query: string;
+}) => {
+  const finalItems = useMemo(() => {
```
Comment (chpa7) [inline play-media-native/hooks/useFacets/useFacets.ts:46]: I'm getting a bit of a strange behavior when filtering using the search input. Even though the items at the top seem correct, there are also some items in the list that do not seem relevant at first sight. Maybe some refinement needed here?

File: play-media-native/helpers/facets.ts [hunk 2]
```diff
@@ -55,3 +57,28 @@ export const getLocationOptions = (events: Event[]) => {
...
+        { label: 'All', value: '' },
+        ...fileTypes.map((fileType) => ({
+          label: fileType,
+          value: fileType,
+        })),
+      ]
+    : [];
+};
+
+export const getActiveSearchFacets = (activeFilters: IIndexable, activeQuery: string) => {
```
[the remark this answers is not in the document]
Comment (sc-nelikostadinova) [reply play-media-native/helpers/facets.ts:79]: activeFilters for me would represent only the filters and not the search. I am open to ideas for naming, searchFacets is the best I could come up with

[the remark this answers is not in the document]
Comment (chpa7) [reply play-media-native/helpers/facets.ts:79]: Yeah just getActiveSearchFacetsNumber then, with 'Number' appended will do

File: play-media-native/features/ListingImages/ListingImages.tsx [hunk 3]
```diff
@@ -33,9 +30,9 @@ const listingStyle = {
   paddingHorizontal: theme.spacing.sm,
 };
 
-const fuseOptions = {
-  keys: ['name'],
-};
+// const fuseOptions = {
```
[the remark this answers is not in the document]
Comment (sc-nelikostadinova) [reply play-media-native/features/ListingImages/ListingImages.tsx:33]: That's fixed in my next PR, if it's not a problem let's leave it to avoid conflicts

**Review by chpa7 — APPROVED** [review]

**Pull request merged**